code/modules/mob/living/carbon/xenomorph/castes/praetorian/castedatum_praetorian.dm

```diff
@@ -101,30 +102,31 @@
 	upgrade = XENO_UPGRADE_TWO
 
 	// *** Melee Attacks *** //
-	melee_damage = 23
+	melee_damage = 19
```

Comment:
Xenos have 2 damage stages, one at young / mature, another at elder / ancient. Increase this to 20.

Reply:
done